pkg/store/store.go

```diff
@@ -236,6 +238,8 @@ func init() {
 	s.NetworkTesterName = "cf-network-tester"
 	s.NetworkTesterGenerateName = "cf-network-tester-"
 	s.NetworkTesterImage = "quay.io/codefresh/cf-venona-network-tester:latest"
+	s.TCPConnectionTesterName = "cf-tcp-connection-tester"
+	s.TCPConnectionTesterImage = "quay.io/codefresh/cf-venona-network-tester:latest"
```

Comment:
why do we need a different tester image, if both the http and tcp network tests use the same one?